packages/cadl-python/tasks.py

```diff
@@ -48,7 +55,26 @@ def _run_cadl(cmds):
   if not success:
     raise SystemExit("Cadl generation fails")
 
-def _run_single_cadl(cmd):
+def prepare_cadl_project(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        exceute_path = args[0][1]
+        testcase = Path(exceute_path).parts[-1]
+        if testcase in CADL_PROJECT:
+            os.chdir(exceute_path)
+            with open("cadl-project.yaml", "w") as file:
```

Comment:
can these flags just be passed on the command line? I think we could do `--option @azure-tools/cadl-python.package-name={}`

This should clean up the code more and make us not need a wrapper

Reply:
@iscai-msft Updated